src/components/Brand.vue

```diff
@@ -1,11 +1,9 @@
 <template>
-  <v-toolbar-title
-    class="wikipedia"
-    v-show="$vuetify.breakpoint.smAndUp"
-    @click.stop="$router.push('/')"
-  >
-    <span class="cap">W</span><span class="smallcap">ikipedi</span
-    ><span class="cap">A</span>
+  <v-toolbar-title class="wikipedia" v-show="$vuetify.breakpoint.smAndUp">
+    <router-link to="/">
```

Comment:
Can be simplified as 
```
<v-toolbar-title 
   class="wikipedia" 
    v-show="$vuetify.breakpoint.smAndUp"
    style="cursor: pointer;"
    @click.stop="$router.push('/')"
>
...
</v-toolbar-title>
```

Reply:
I saw the click handler before, but with that there are problems :
* If user is on page '/' and click it again, this error throws up in console :
```
Error in v-on handler (Promise/async): "Error: Avoided redundant navigation to current location: "/"."
```
* If a user wants to open wikivue in a new tab or window or incognito, they have to manually type the url and do it, having the brand as a hyperlink, the user will be easily to do this
* As for theming, I'll avoid using `!important` and use a class which will help theming

Reply:
This is how this patch looks like in dark theme: ![image](https://user-images.githubusercontent.com/161672/88901540-8c483780-d26e-11ea-97a7-83164fcf732e.png)

Expected result: 
![image](https://user-images.githubusercontent.com/161672/88902339-b2baa280-d26f-11ea-8d3c-5cd25692961d.png)

Dark theme can be selected from settings or follows browser preferences. 

Wikivue does not use `#000` as color anywhere. It is not a natural color. The dark color is `rgba(0,0,0,.87)`

The error in "$router.push('/')" when duplicate navigation happens can be either ignored or can be fixed using `this.$router.push("/").catch(()=>{});

Whether we should support opening the same application in new tab by clicking the brand component - If we want to support that, we should use a Vuetify button or link component to do that. That will follow the theming and styling. We should not create custom styled links - that defeat the purpose of UI library.

Reply:
It's my first time with Vuetify, didn't know that. Thanks! I've made it into a `v-btn` now, the significant change now is that it has a background if user is on `/`, is that okay ?
![image](https://user-images.githubusercontent.com/2305675/88909169-6889ef00-d278-11ea-9bb8-126425adf27a.png)

Reply:
It is bit tricky to use the various flags available and documented https://vuetifyjs.com/en/components/buttons/ to get desired rendering. 

Anyway, let us not make the existing UI bad for achieving the usecase you mentioned. The current size is not ok and we don't need background if user is in `/`. 

There should be some non-hacky way to do this, but I did not explore it. If there is a common pattern using vuetify, we can use the same in smc.org.in site as well :-)

Reply:
Vuetify docs has the hyperlink for their brand logo, so I peeked into their code and found that they're using `router-link` plus different SVG logo file for dark & light themes : https://github.com/vuetifyjs/docs-next/blob/48ca2833a7a8379bec8a432cdc86d57cec184f48/src/components/vuetify/Logo.vue#L2

So I'm gonna do this now.

Reply:
So the solution was very easy, only needed to add two classes !